README.md

```diff
@@ -1,3 +1,94 @@
 # xdggs
 
-Xarray extension for DGGS
+`xdggs` is an extension for [Xarray](https://xarray.pydata.org/) that provides tools to handle geospatial data using Discrete Global Grid Systems (DGGS). It allows efficient manipulation and analysis of multi-dimensional gridded data within a DGGS framework, facilitating spatial data processing, resampling, and aggregations on both global and regional scales.
+
+## Key Features
+
+- **Seamless Integration with Xarray**: Use `xdggs` alongside Xarray's powerful tools for managing labeled, multi-dimensional data.
+- **Support for DGGS**: Convert geospatial data into DGGS representations, allowing for uniform spatial partitioning of the Earth's surface.
+- **Spatial Resampling**: Resample data on DGGS grids, enabling downscaling or upscaling across multiple resolutions.
+- **DGGS Aggregation**: Perform spatial aggregation of data on DGGS cells.
+- **Efficient Data Management**: Manage large datasets with Xarray's lazy loading, Dask integration, and chunking to optimize performance.
+
+## Installation
+
+To install `xdggs`, you can clone the repository and install it using pip:
+
+```bash
+git clone https://github.com/your-repo/xdggs.git
+cd xdggs
+pip install .
+```
+
+Alternatively, you can install it directly via pip (once it's available on PyPI):
+
+```bash
+pip install xdggs
+```
+
+## Demo
+
+![xdggs demo](xdggs-cropped.gif)
+
+## Getting Started
+
+Here's a quick example of how to use `xdggs` to convert your Xarray data to a DGGS grid and perform resampling.
+
+```python
+import xarray as xr
+import xdggs
+
+# Load your Xarray dataset
+ds = xr.open_dataset("data/h3.nc", engine="netcdf4")
+ds_idx = ds.pipe(xdggs.decode)
+ds_idx.dggs.sel_latlon(np.array([37.0, 37.5]), np.array([299.3, 299.5]))
```

Comment:
the result of this operation is immediately discarded. Since this can be misunderstood as an inplace operation, can we either remove this or assign the result to a variable and make use of it?

Reply:
I just made a copy/paste from the first example I found but I think we should put a more relevant example. What would be better to have here?

Reply:
not sure. The most important parts of `xdggs` right now are:
- decoding / creating the index
- selecting by geographical coordinates without actually having the geographical coordinates on the object (`sel_latlon`)
- extracting cell center coordinates from the cell ids (`cell_centers`, and the older `assign_latlon_coords`)
- extracting cell boundaries from the cell ids (`cell_boundaries`)
- plotting with `lonboard` (`explore`)

Reply:
I updated the readme.  I concentrated the example only for loading file, assigns coordinate, plotting.  More detailed part can be in the documentation, other example files.  
The essential needs from simpler / first usage is that you can load and plot (and in the same interface for H3 & Healpix &....